Formula/tcc.rb

```diff
@@ -27,6 +26,11 @@ class Tcc < Formula
     sha256 x86_64_linux: "053f79a5752554e18ecba168184e48481bce8a2db418a3f9b0de094f9e6d0e4d"
   end
 
+  head do
+    url "https://repo.or.cz/tinycc.git", branch: "mob"
+    mirror "https://github.com/TinyCC/tinycc.git"
```

Comment:
So this does get rid of the error, but unfortunately `brew` doesn't fall back to the mirror if cloning from the `url` fails.

We could probably just use the GitHub URL for `head` instead.

Reply:
We could maybe add `mirror` support for `head` in `brew`, depending on how complicated it turns out to be. If it's a small change then we'd probably merge a pull request.

Reply:
I’ll look into it when I next get the chance.  It seems like it might be useful beyond this formula anyway.